### PR TITLE
Fixing some typos

### PR DESCRIPTION
**Pull request by 4c74356b41**
It’s been almost a year since we announced the support for Virtual Machines under Azure Resource Manager. You can read more about the advancements and the additional capabilities that it supports >>>here<<< no link. since I'm not sure what link do you want i'm not inserting any
And there’s a focus of enabling security by default with the enforcement of virtual machines in a Virtual Network. - shouldn't it be "focus ON"?

I corrected a few is\are here and there, not really sure i'm 100% correct, please check. I'm not a native English speaker. Thanks!

Also, I've edited "Resource Manager" and "Virtual Machine" to always start from a capital letter, since you mostly had it capitalized, but not everywhere

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

File: articles/virtual-machines/virtual-machines-windows-migration-classic-resource-manager.md [hunk 1]
```diff
@@ -19,11 +19,11 @@
...
+We are really excited about the power and capability offered by the new experience and APIs available on Virtual Machines. We think this is really going to change the way you can use the cloud in a lot of ways. With the release of the new model, it allows you to deploy, manage, and monitor related services in a resource group. Resource Manager enables deploying complex applications using templates, configures Virtual Machines using VM extensions, and incorporates access management and tagging. It also includes scalable, parallel deployment for Virtual Machines into availability sets. In addition, the new model provides lifecycle management of Compute, Network and Storage independently. And there’s a focus of enabling security by default with the enforcement of virtual machines in a Virtual Network.
```
Comment (4c74356b41) [inline articles/virtual-machines/virtual-machines-windows-migration-classic-resource-manager.md]: virtual machine to Virtual Machine

File: articles/virtual-machines/virtual-machines-windows-migration-classic-resource-manager.md [hunk 2]
```diff
@@ -40,29 +40,29 @@ Please note that one of the important changes that you will have to make as part
...
-- Data Plane (Application) – This describes the “runtime” of the application itself and involve interaction with instances that don’t go through the Azure API. Accessing your website or pulling data from a running SQL Server or mongoDB server would all be considered data plane or application interaction. Copying a blob from a storage account and accessing a Public IP address to RDP or SSH into the Virtual Machine also are data plane. These operations keep the application running across compute, networking, and storage.
+- Data Plane (Application) – This describes the “runtime” of the application itself and involve interaction with instances that don’t go through the Azure API. Accessing your website or pulling data from a running SQL Server or MongoDB server would all be considered data plane or application interaction. Copying a blob from a storage account and accessing a Public IP address to RDP or SSH into the Virtual Machine also are data plane. These operations keep the application running across compute, networking, and storage.
```
Comment (4c74356b41) [inline articles/virtual-machines/virtual-machines-windows-migration-classic-resource-manager.md]: mongoDB to MongoDB

**Comment by tysonn** [comment]
@mahthi - Please review this public repo PR from @4c74356b41

**Comment by mahthi** [comment]
Looks good to me! Thanks!

**Comment by 4c74356b41** [comment]
Can you please look into this?\

It’s been almost a year since we announced the support for Virtual Machines under Azure Resource Manager. You can read more about the advancements and the additional capabilities that it supports >>>here<<< no link. since I'm not sure what link do you want i'm not inserting any
And there’s a focus of enabling security by default with the enforcement of virtual machines in a Virtual Network. - shouldn't it be "focus ON"?

**Comment by mahthi** [comment]
It’s been almost a year since we [announced support for Virtual Machines under Azure Resource Manager.](http://azure.microsoft.com/blog/2015/04/29/iaas-just-got-easier-again/) You can read more about the [advancements ](http://azure.microsoft.com/en-us/documentation/articles/virtual-machines-azurerm-versus-azuresm/)and the [additional capabilities](http://azure.microsoft.com/blog/2015/07/30/tagging-on-compute-network-and-storage-resources/) of the new stack. We also gave [guidance ](http://azure.microsoft.com/en-us/documentation/articles/virtual-networks-arm-asm-s2s-howto/)around how to best connect and have resources from the two deployment models co-exist in your subscription using Virtual Network Site-Site Gateways. In addition, we released the capability to [configure your ExpressRoute circuit to connect to both your classic and resource manager virtual networks](http://azure.microsoft.com/en-us/documentation/articles/expressroute-move/).

You can update it with the above text to make it cleaner.

**Comment by 4c74356b41** [comment]
I'm sorry, I'm kinda new to this, how do I incorporate that into my PR and keep links working?

**Comment by tysonn** [comment]
@mahthi - I'd recommend you submit the update in a pull request against @4c74356b41 fork/branch. Then, all @4c74356b41 has to do is merge your pull request.

**Comment by 4c74356b41** [comment]
this stuff now says that it has conflicts that must be resolved =\

**Comment by 4c74356b41** [comment]
Well, are you folks actually going to do anything about those 2 fixes or not?

**Comment by mahthi** [comment]
Its in my backlog and I'll get to it next week.

**Comment by tysonn** [comment]
I have taken care of this. @4c74356b41, thanks for the updates.

**Pull request merged**